scripts/monit_start_service.py

```diff
@@ -50,9 +64,19 @@ def start_service(service_name):
     logging.error("Monit was unable to start " + service_name)
     return 1
   else:
+    sleep_until_service_is_up(service_name)
     logging.info("Successfully started " + service_name)
     return 0
 
+def sleep_until_service_is_up(service_name):
```

Comment:
Can you pass the subprocess command and args as params to this function?
That way the body of this function could be simplified and look something like:

```
while subprocess.call([command, cmd_args]) != 0:
  time.sleep(10)
```